mne_bids/tests/test_write.py

```diff
@@ -2187,8 +2187,8 @@ def test_event_storage():
 @pytest.mark.parametrize(
     'dir_name, fname, reader, datatype, coord_frame', [
         ('EDF', 'test_reduced.edf', _read_raw_edf, 'ieeg', 'mni_tal'),
-        ('EDF', 'test_reduced.edf', _read_raw_edf, 'ieeg', 'fs_tal'),
-        ('EDF', 'test_reduced.edf', _read_raw_edf, 'ieeg', 'unknown'),
+        # ('EDF', 'test_reduced.edf', _read_raw_edf, 'ieeg', 'fs_tal'),  # XXX: shouldn't be used  # noqa
```

Comment:
Why not?

Reply:
Because of https://github.com/mne-tools/mne-bids/pull/704#discussion_r583889948

But I can add this back in I suppose and just have this set to "other"?

Reply:
Oh the "shouldn't be used" is referring to `fs_tal`, not to the actual test itself.

Well… why don't you simply remove this entire line, then?